lib/build.sh

```diff
@@ -150,8 +150,8 @@ pushd "${CHECKOUT}/libvips-${VIPS_VERSION}-${TARGET}"
      --enable-shared --disable-static \
      --with-jpeg-includes=$PREFIX/include \
      --with-jpeg-libraries=$PREFIX/lib \
-     --with-png-includes=$PREFIX/include \
-     --with-png-libraries=$PREFIX/lib \
+     PNG_CFLAGS="-I$PREFIX/include/ -I$PREFIX/include/libpng16/" \
+     PNG_LIBS="-L$PREFIX/lib/ -lpng16" \
```

Comment:
Why png16? I've never seen it before.

Reply:
We use libpng 1.6.34 and I want to override pkg-config linker / compiler flags.
On travis ubuntu xenial container, you have a pre-installed libpng12 which conflict with our one.